src/main/java/org/opensearch/ad/ml/EntityColdStarter.java

```diff
@@ -331,9 +334,14 @@ private void trainModelFromDataSegments(List<double[][]> dataPoints, Entity enti
             .dimensions(rcfNumFeatures)
             .sampleSize(rcfSampleSize)
             .numberOfTrees(numberOfTrees)
-            .lambda(rcfTimeDecay)
+            .timeDecay(rcfTimeDecay)
             .outputAfter(numMinSamples)
             .parallelExecutionEnabled(false)
+            .compact(true)
+            .precision(Precision.FLOAT_32)
+            .boundingBoxCacheFraction(AnomalyDetectorSettings.REAL_TIME_BOUNDING_BOX_CACHE_RATIO)
+            // same with dimension for opportunistic memory saving
+            .shingleSize(rcfNumFeatures)
```

Comment:
I think it should be ShingleSize?

Reply:
that is ".shingleSize(shingleSize)"

Reply:
changed.

Reply:
Not seeing change though ... should I refresh/do something else?